actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -88,13 +88,13 @@ def each(&block)
 
     def self.return_only_media_type_on_content_type=(*)
       ActiveSupport::Deprecation.warn(
-        ".return_only_media_type_on_content_type= is dreprecated with no replacement and will be removed in 7.0."
+        ".return_only_media_type_on_content_type= is deprecated with no replacement and will be removed in 6.2."
```

Comment:
The 7.0 -> 6.2 change is wrong, we don't have plans to release a 6.2 version.

Reply:
Thank you for letting me know, and updating it.